lms/static/sass/_build-course.scss

```diff
@@ -33,6 +33,7 @@
 @import 'course/modules/student-notes'; // student notes
 @import 'course/modules/calculator'; // calculator utility
 @import 'course/modules/timer'; // timer
+@import 'course/image-zoom-tool'; // styles for the custom image zoom tool
```

Comment:
Assuming we stick with this lms/cms duplication (uck), put the sass file under course/modules/html.